specification/purview/data-plane/readme.md

```diff
@@ -70,7 +70,7 @@ csharp:
   license-header: MICROSOFT_MIT_NO_VERSION
   namespace: Microsoft.Azure.Purview.CatalogClient
```

Comment:
Isn't this the namespace?

Reply:
This PR only renames Azure.Purview.Catalog to Azure.Analytics.Purview.Catalog as well as some rollbacks.